application/gqlgen/schema.graphql

```diff
@@ -65,14 +65,27 @@ type User {
     organizations: [Organization!]!
     posts(role: PostRole!): [Post!]!
     photoURL: String!
+    preferences: [UserPreference!]
```

Comment:
```suggestion
    preferences: [UserPreference!]!
```
You may have a reason for this, but I have yet to see a case where a list is not always provided.

Reply:
ok